overlay.d/05rhcos/usr/lib/dracut/modules.d/41rhcos-check-luks-syntax/rhcos-fail-boot-for-older-luks-syntax.service

```diff
@@ -0,0 +1,15 @@
+# Fixes the encryption issue with LUKS in 4.7 when a user 
+# tries to use the RHCOS 4.6 or earlier LUKS syntax in an 
+# Ignition config.
+[Unit]
+Description=RHCOS Check For Legacy LUKS Configuration 
+Before=ignition-disks.service
+After=ignition-fetch.service
```

Comment:
I think it's clearer to list `After` dependencies first so the dependencies are in chronological order.